hbase/src/main/java/org/apache/metamodel/hbase/HBaseDataContext.java

```diff
@@ -222,9 +222,11 @@ protected DataSet materializeMainSchemaTable(Table table, List<Column> columns,
         for (Column column : columns) {
             if (!column.isPrimaryKey()) {
                 final int colonIndex = column.getName().indexOf(':');
+                final int len = column.getName().length();
                 if (colonIndex != -1) {
                     String family = column.getName().substring(0, colonIndex);
-                    scan.addFamily(family.getBytes());
+                    String colName = column.getName().substring(colonIndex + 1, len);
+                    scan.addColumn(family.getBytes(),colName.getBytes());
```

Comment:
... and a space after the comma :-)

Reply:
Ok,I will modify the code at once and thanks Peter kaspersorensen for reviewing.